### PR TITLE
Update tyme-3-time-tracker extension

### PR DESCRIPTION
**Pull request by k-and**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/tyme-3-time-tracker/src/start-task-tracking.tsx [hunk 1]
```diff
@@ -1,7 +1,11 @@
-import { Action, ActionPanel, Icon, List, closeMainWindow, showHUD } from "@raycast/api";
-import { Task, getTasks, startTrackingTask } from "./tasks";
+import { Action, ActionPanel, Icon, List, closeMainWindow, showHUD, getPreferenceValues } from "@raycast/api";
+import { Task, getTasks, startTrackingTask, stopTracking } from "./tasks";
 import { useCachedPromise, useFrecencySorting } from "@raycast/utils";
 
+interface Preferences {
+  stopRunningTimers: boolean;
```
Comment (pernielsentikaer) [inline extensions/tyme-3-time-tracker/src/start-task-tracking.tsx:6]: You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/tyme-3-time-tracker/src/start-task-tracking.tsx (hunk 1 above)
Comment (k-and) [reply extensions/tyme-3-time-tracker/src/start-task-tracking.tsx]: Hey, thanks for the review. I believe it's implemented properly now.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**